docs/source/sql.rst

```diff
@@ -37,3 +37,16 @@ These are often an order of magnitude faster than the ``Python->SQLAlchemy``
 route when they are available.::
 
     sqlalchemy.Table <- CSV
+
+Enabling Conversions
+--------------------
+
+Odo is dependent on the SQLAlchemy library for sqlite, PostgreSQL, MySQL, Hive,
+and RedShift support.  SQLAlchemy is not installed by default when Odo is
+installed, therefore you must install it if you want to use one of these
+databases.
+
+If SQLAlchemy is not installed and you try using one of the databases, you
+may get the following error::
```

Comment:
What do you think about making a general statement about enabling conversions, and then giving SQLAlchemy as a specific example.

I'd also like to include a shell command for installation, for completeness.

```sh
$ conda install sqlalchemy
$ pip install sqlalchemy
```